src/org/opendatakit/briefcase/model/BriefcasePreferences.java

```diff
@@ -143,4 +156,17 @@ public Preferences preferenceFactory(Class<?> node) {
     private static final BriefcasePreferences APPLICATION_SCOPED =
         new BriefcasePreferences(BriefcasePreferences.class, PreferenceScope.APPLICATION);
   }
+  
+  public static String getBriefCaseProxyType() {
```

Comment:
For the sake of a symmetric interface, could we remove the below three static methods and refactor into one method, `public static ProxyConnection getBriefcaseProxyProperty()`?  Also the naming here feels a little smurfed to me (but I don't feel strongly).  I think "briefcase directory" makes sense, but these could probably just be "getProxyProperty" and "setProxyProperty."